.github/workflows/acceptance.yaml

```
@@ -131,6 +131,8 @@ jobs:
           web3:
             env:
               HIERO_MIRROR_WEB3_OPCODE_TRACER_ENABLED: "true"
+              HEDERA_MIRROR_WEB3_EVM_MODULARIZEDSERVICES: "true"
```

We probably want to revert these changes and just let it fail for now.

The images used in acceptance tests are not built from the PR branch and as a result if any changes in the PR branch have an impact on acceptance tests, it'll just fail and should fail.

Once we get a 0.139.0-rc1 tag and update the version here, it will pass.